scripts/us_epa/superfund/site_contamination/process_sites_contamination.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contaminant of concern dataset for superfund sites import
+
+This import uses the dataset:
+- ./data/401062.xlsx
+    This file describes Contaminant of concern data from Superfund decision documents issued in fiscal years 1982-2017. Includes sites 1) final or deleted on the National Priorities List (NPL); and 2) sites with a Superfund Alternative Approach (SAA) Agreement in place. The only sites included that are 1) not on the NPL; 2) proposed for NPL; or 3) removed from proposed NPL, are those with an SAA Agreement in place.
+"""
+
+from absl import app, flags
+import os
+import sys
+import pandas as pd
+
+# Allows the following module imports to work when running as a script
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../..'))  # for utils
+from us_epa.util.superfund_vars import _CONTAMINATED_THING_DCID_MAP, _CHEM_MAP
+
+sys.path.append(os.path.join(_SCRIPT_PATH,
+                             '../../../../util/'))  # for statvar_dcid_generator
+from statvar_dcid_generator import get_statvar_dcid
+
+_TEMPLATE_MCF = """
+Node: E:SuperfundSite->E0
+typeOf: dcs:StatVarObservation
+observationAbout: C:SuperfundSite->observationAbout
+observationDate: C:SuperfundSite->observationDate
+variableMeasured: C:SuperfundSite->variableMeasured
+value: C:SuperfundSite->value
+"""
+
+_DATASET_NAME = "./401062.xlsx"
+_COL_NAME_MAP = {
+    'EPA ID': 'observationAbout',
+    'Actual Completion Date': 'observationDate'
+}
+
+
+def make_contamination_svobs(df: pd.DataFrame,
+                             output_path: str) -> pd.DataFrame:
+    """
+    Function makes SVObs of contaminated medium at the site concatenated with '&' as the observed value.
+    """
+    df = df.drop_duplicates()
+    # there are some rows where contaminatedThing is nan, which we drop
+    df = df.dropna()
+    df['Media'] = df['Media'].apply(lambda x: _CONTAMINATED_THING_DCID_MAP[x])
+    df = df.groupby(['EPA ID', 'Actual Completion Date'],
+                    as_index=False)['Media'].apply('&'.join).reset_index()
+    # The groupby is handled differently in Python3.7 and Pytohn3.9, hence we have this check. In python3.9, the column is preserved but in 3.7 the column name in groupby is replaced with 0
```

Comment:
```suggestion
    # The groupby is handled differently in Python3.7 and Python3.9, hence we have this check. In python3.9, the column is preserved but in 3.7 the column name in groupby is replaced with 0
```

Reply:
this is a nice catch -- thanks for documenting.

Reply:
It was fun getting to this, @ajaits  mentioned that the tests were failing because of the environment and after testing each part of the code mimicking the cloud build environment we learnt of this subtle implementation issue. But I think I will need to rephrase this comment because it is not the groupby method of Python but that of pandas, will update the comment.

Reply:
ok, please update. i also have a [PR](https://github.com/datacommonsorg/data/pull/613) that i'm about to merge which will update the packages -- please check that the behavior remains as expected. that pr is only possible because of end-to-end tests, which is why we keep asking for them :)

Reply:
Yes, I have updated the comment in the latest commit [3fb7a665](https://github.com/datacommonsorg/data/pull/586/commits/3fb7a6652e5b7b8e87d502ef25cf20dd87c1d64d), and the cloud build tests pass. I have tested the scripts in this PR locally with Python 3.9 and the upgraded `pandas` version and the tests pass for them too.